tests/compat/test_porcelain.py

```diff
@@ -91,3 +91,54 @@ def test_verify(self):
         tag = self.repo[b"refs/tags/verifyme"]
         self.assertNotEqual(tag.signature, None)
         tag.verify()
+
+
+@skipIf(
+    platform.python_implementation() == "PyPy" or sys.platform == "win32",
```

Comment:
please make this dependent on gpgme actually being importable. Even on linux people might not have it installed and if we do manage to install gpgme on mac or win32 in the ci, we wouldn't have to manually update this code.

Reply:
I believe it is. CommitCreateSignTestCase inherits from PorcelainGpgTestCase which has a decorator
```
@skipIf(gpg is None, "gpg is not available")
```
in `dulwich/tests/test_porcelain.py`.

I don't 100% understand these `@skipIf` decorators but I suspect the current code is doing what you want.